lib/proxy_api/external_ipam.rb

```diff
@@ -44,30 +44,28 @@ def initialize(args)
     #   }
     # }
     #
-    # Params: 1. subnet:  The IPv4 or IPv6 subnet CIDR. (Examples: IPv4 - "100.10.10.0/24", IPv6 - "2001:db8:abcd:12::/124")
-    #         2. mac:     The mac address of the interface obtaining the IP address for
-    #         3. group:   The group in External IPAM that the subnet belongs to(e.g. This would be "Section" in phpIPAM)
+    # Params: 1. subnet:           The IPv4 or IPv6 subnet CIDR. (Examples: IPv4 - "100.10.10.0/24", IPv6 - "2001:db8:abcd:12::/124")
+    #         2. mac:              The mac address of the interface obtaining the IP address for
+    #         3. group(optional):  The group in External IPAM that the subnet belongs to (e.g. 'Section' in
+    #                              in phpIPAM, or 'VRF' in Netbox)
     #
-    # Returns: A hash with the next available IP in the "data" field, or a hash with "error" key
-    #          containing error message
-    #
-    # Examples:
+    # Responses:
     #   Responses if success:
-    #     IPv4: {"data": "100.55.55.3"}
-    #     IPv6: {"data": "2001:db8:abcd:12::1"}
+    #     "100.55.55.3"          (IPv4)
+    #     "2001:db8:abcd:12::1"  (IPv6)
```

Comment:
This is actually good practice, having root element enables you to extend the API without breaking clients! I highly suggest the opposite - to review all calls to actually have a root element called "data" ;-)

I mentioned this in your smart proxy PR the other day.